### PR TITLE
Support deploying to AWS

### PR DESCRIPTION
**Pull request by chrisvire**
Update http://dev.scriptoria.io with latest build from master.  

- [x] Build release docker images
- [x] Push images to AWS ECR
- [x] Update AWS ECS to use new images
- [x] On startup, optionally bootstrap the database (`DB_BOOTSTRAP=1`) and load sample data (`DB_SAMPLEDATA=1`)

When we separate our work into master (for production) and develop (for staging), then uncomment code in `scripts/deploy-travis.sh`.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -46,7 +49,7 @@ jobs:
 
     - stage: "Deploy"
       name: "Build and Push Docker Images"
-      if: branch = master
+      if: branch IN (develop, master) OR (branch =~ /^deploy-.*$/)
```
Comment (NullVoxPopuli) [inline .travis.yml]: what are examples of `deploy-`? branches?
(I'm mostly familiar with gitflow, which uses `release/version-stuff` -- is `deploy-` similar?

File: .travis.yml [hunk 2]
```diff
@@ -1,6 +1,9 @@
 env:
   - DOCKER_COMPOSE_VERSION=1.17.0 MOZ_HEADLESS=1
 
+stages:
+  - "Tests"
```
Comment (NullVoxPopuli) [inline .travis.yml]: If we can find a linter for the C# code, I'd like to add a pre-test stage for "Quality". But that can happen later

File: .travis.yml (hunk 1 above)
Comment (chrisvire) [reply .travis.yml]: I just came up with another branch prefix for my testing (I was using deploy-aws).  I can remove that.

Comment (NullVoxPopuli) [reply .travis.yml]: it's fine -- just depends on what you want your process to be
in gitflow:
```
 - master                -> production
 - release/*             -> staging 
 - develop               -> ci
 - feature/* or bugfix/* -> review deploy
```

**Pull request merged**